### PR TITLE
kinetic-client 0.0.6

### PR DESCRIPTION
**Pull request by toolslive**

### CODE REVIEW

File: packages/kinetic-client/kinetic-client.0.0.6/opam [hunk 1]
```diff
@@ -0,0 +1,27 @@
+opam-version: "1.2"
+name: "kinetic-client"
+version: "0.0.6"
+maintainer: "Romain Slootmaekers <romain.slootmaekers@cloudfounders.com>"
+authors: "Romain Slootmaekers <romain.slootmaekers@cloudfounders.com>"
+homepage: "https://github.com/cloudfounders/kinetic-ocaml-client"
+bug-reports: "https://github.com/cloudfounders/kinetic-ocaml-client"
+dev-repo: "https://github.com/cloudfounders/kinetic-ocaml-client"
```
Comment (dsheets) [inline packages/kinetic-client/kinetic-client.0.0.6/opam]: I think this should have `.git` on the end so opam has a chance to figure out the VCS protocol.

Comment (toolslive) [inline packages/kinetic-client/kinetic-client.0.0.6/opam]: the manual is fuzzy (also contains a typo there)
> dev-repo: <string>: the URL of the package's source repository, which may be useful for developpers: not to be  > mistaken with the URL file, which points to the specific packaged version.

and opam lint accepts it. Anyway, I changed it.

Comment (dsheets) [inline packages/kinetic-client/kinetic-client.0.0.6/opam]: Yes, I will open issues for both these problems. It's certainly not your fault. Thanks for your attention to detail. :-)

**Comment by dsheets** [comment]
The VCS issue is now tracked by ocaml/opam#2240.

**Pull request merged**